pkg/controller/storagecluster/reconcile.go

```diff
@@ -739,23 +738,8 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string) *cephv1.CephClus
 		},
 	}
 
-	// If a MonPVCTemplate is provided, use that. If not, if StorageDeviceSets
-	// have been provided, use the StorageClass of the DataPVCTemplate from the
-	// first StorageDeviceSet for providing the Mon PVs
 	if sc.Spec.MonPVCTemplate != nil {
 		cephCluster.Spec.Mon.VolumeClaimTemplate = sc.Spec.MonPVCTemplate
-	} else if len(sc.Spec.StorageDeviceSets) > 0 {
-		ds := sc.Spec.StorageDeviceSets[0]
-		cephCluster.Spec.Mon.VolumeClaimTemplate = &corev1.PersistentVolumeClaim{
```

Comment:
Is simply not specifying a PVC template sufficient to tell Rook-Ceph to use hostPath, or is other configuration required?

Reply:
I think is sufficient